### ISSUE TITLE
Q&A

### ISSUE THREAD

**Issue by alinazeng** [lizzieinvancouver/localadaptclim#5]
Dear Lizzie @lizzieinvancouver ,

Do not be sorry! I am so glad you will finally be able to travel and meet with your collaborators!

And yes, I hope Mira is handling the final stage of her research well. Please tell her I am sending her good thoughts when you see her.

I currently have 11-12 datasets with data that we need (latitude and elevation of common gardens, latitude and elevation of provenances, DOY of spring events, year) 

> about 4 European ones and 8 North American ones. Will need to look for more European studies later.

I have not yet formatted all datasets into one giant dataset but will do so after I gather the climate data. For now, I have everything in separate tabs (please see attached a subset of the data I have at the moment). 

[LocalAdaptionDataGathering.xlsx](https://github.com/lizzieinvancouver/localadaptclim/files/7240692/LocalAdaptionDataGathering.xlsx)

Here is a list of questions for you. I hope they do not take away too much of your time:

1) There is a New Zealand study. Do I exclude it?

2) There are a few controlled experiments done in the greenhouse. Do I exclude those ones? They look at how temperature and chilling play a part in the spring events of plants of different provenances, I am not sure if we want those cuz the environment is controlled and not affected by climate.

3) Some studies have days to bb rather than DOY. The reference date is when they start the control ... this circles back to bullet point 2: do we include controlled experiments?

4) If a study looks at trees (from North American provenances) planted in Europe, do we consider them as North American studies or European ones? 

5) Could you let me know if this graph (attached) can be of any use? The triangles represent different provenances, but there is no way to tell which triangle is which provenance :')))

![image](https://user-images.githubusercontent.com/83845945/132731033-9e328371-4737-4941-afab-70d0c67525a9.png)

6) This[ Liu & El-Kassaby 2019 paper](https://github.com/lizzieinvancouver/localadaptclim/blob/main/data/journal_artilces_found_online/NA%20Liu%20El-Kassaby%202019.pdf) has data for canopy duration (from budburst to leaf drop), do you think we can use this or?

7) Some studies do not mention the year when spring event was observed. Could we still work with those?

8) Some studies do not mention the longitude or elevation of the gardens. Some do not even mention the latitude. Could we still work with those?

9) question about next step: I can start collecting climate data now and look for more datasets if needed later. Could you remind me where I could look for climate data again? am I gathering daily temperature for each observation according to the year/doy/latitude/longitude of both the gardens and the provenances? how do you envison the data to be formatted (is it going to be **`lat_garden|long_garden|elev_garden|lat_provenance|long_provenance|elev_provenance|species|year|DOY|daily_temperature`**?) Please let me know.

Thank you so much for bearing with such a long list of things. Please take your time! I appreciate you Lizzie.

Warmly,
Alina

**Comment by lizzieinvancouver** [comment]
@alinazeng I am so sorry I dropped the ball on issue #4 ... feel to add a '@lizzieinvancouver, still waiting on this note' to git issues I miss. They sometimes get lost in a flood of issues.

Anyway. Onwards!

The thing we have to consider in answering many of these questions is the questions we want to answer. I have them in my head as:

1. Is local adaptation in spring phenology stronger in Europe versus North America? (Relatedly, is there a difference between conifers and angiosperms)?
2. Is the magnitude of local adaptation predictable related to variation in spatial and temporal climate? 

Okay, so with this in mind my answers to your queries are:

1. Exclude New Zealand.
2. I think we can try to include these! So let's include data from field and indoor-experiment studies. 
3. Yes! We can work to use all different date formats; as long as they are in some unit of 'day' then we can get around difference reference points. We should add a note for each study though on what the 'day' is relative to.
4. Wow! We consider them as North American trees ... we can definitely use them to answer our first question, but maybe not our second (I am not sure, the second question is harder to answer).
5. I think no -- we could address our first question without exact provenance data, but we still need a way to estimate local adaptation and I don't see how to do that with that figure. 
6. We need budburst or leafout data, so if they only provide duration, then we cannot use it.
7. Yes! 
8. Hmm ... for question 1, we could maybe use any data, but it will be tricky to have good estimates of local adaptation without ANY way to tell how far apart the gardens are. So ... I think they must provide either latitude or elevation; if they do not give either of those we cannot use it.
9. I will make a new issue for this as it is a lot of work. See issue #6
